module/v1/schema.json

```diff
@@ -0,0 +1,318 @@
+        ".*": {
+          "$ref": "#/definitions/tupleParameter"
+        }
+      }
+    },
+    "topics": {
+      "type": "object",
+      "description": "Topics of the module",
+      "patternProperties": {
+        ".*": {
+          "$ref": "#/definitions/tupleParameter"
+        }
+      }
+    },
+    "tools": {
+      "type": "array",
+      "description": "Tools used by the module",
+      "items": {
+        "type": "object",
+        "patternProperties": {
+          ".*": {
+            "type": "object",
+            "properties": {
+              "description": {
+                "type": "string",
+                "description": "Description of the tool"
+              },
+              "homepage": {
+                "type": "string",
+                "description": "Homepage of the tool",
+                "pattern": "^https?://.*$"
+              },
+              "documentation": {
+                "type": "string",
+                "description": "Documentation of the tool",
+                "pattern": "^(https?|ftp)://.*$"
+              },
+              "tool_dev_url": {
+                "type": "string",
+                "description": "URL of the development version of the tool's documentation",
+                "pattern": "^https?://.*$"
+              },
+              "doi": {
+                "description": "DOI of the tool",
+                "anyOf": [
+                  {
+                    "type": "string",
+                    "pattern": "^10\\.\\d{4,9}\\/[^,]+$"
+                  },
+                  {
+                    "type": "string",
+                    "enum": ["no DOI available"]
+                  }
+                ]
+              },
+              "licence": {
+                "type": "array",
+                "items": {
+                  "type": "string"
+                },
+                "description": "Licence of the tool",
+                "minItems": 1,
+                "uniqueItems": true
+              },
+              "identifier": {
+                "type": "string",
+                "description": "bio.tools identifier of the tool",
+                "pattern": "^(biotools:.*)?$"
+              }
+            },
+            "required": ["description"],
+            "anyOf": [
+              {
+                "required": ["homepage"]
+              },
+              {
+                "required": ["documentation"]
+              },
+              {
+                "required": ["tool_dev_url"]
+              },
+              {
+                "required": ["doi"]
+              }
+            ]
+          }
+        }
+      }
+    },
+    "extra_args": {
+      "type": "array",
+      "description": "Extra arguments for the module",
+      "items": {
+        "type": "object",
+        "properties": {
+          "description": {
+            "type": "string",
+            "description": "Description of the argument"
+          }
+        }
+      }
+    },
```

Comment:
I couldn't find any examples of this in nf-core/modules. Is it related to the `task.ext` properties?